core/runtime.rs

```diff
@@ -2294,60 +2334,6 @@ pub mod tests {
       .unwrap();
   }
 
-  #[test]
-  fn test_ref_unref_ops() {
```

Comment:
FYI op_test is resolved immediately hence this ref and unref doesn't make sense.

Reply:
added back as deferred op